nikola/plugins/task/indexes.py

```diff
@@ -296,3 +335,12 @@ def index_section_atom_path(self, name, lang):
         link://section_index_atom/cars => /cars/index.atom
         """
         return self.index_section_path(name, lang, is_feed=True)
+
+    def index_section_rss_path(self, name, lang):
+        """Link to the RSS feed for a section.
+
+        Example:
+
+        link://section_index_atom/cars => /cars/rss.xml
```

Comment:
Yep.

El dom., ago. 21, 2016 14:45, Chris Warrick <notifications@github.com>
escribió:

> In nikola/plugins/task/indexes.py
> <https://github.com/getnikola/nikola/pull/2463#discussion_r75599839>:
>
> > @@ -296,3 +335,12 @@ def index_section_atom_path(self, name, lang):
> >          link://section_index_atom/cars => /cars/index.atom
> >          """
> >          return self.index_section_path(name, lang, is_feed=True)
> > +
> > +    def index_section_rss_path(self, name, lang):
> > +        """Link to the RSS feed for a section.
> > +
> > +        Example:
> > +
> > +        link://section_index_atom/cars => /cars/rss.xml
>
> That’s supposed to be section_index_rss, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/getnikola/nikola/pull/2463/files/1ccfadec9df7ed57f39c36f4525ade1243922b36#r75599839>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAGKzT1BUtMUf4bjcsdRZVy6QiaLZR2ks5qiI6dgaJpZM4Jo-MQ>
> .
>